selftests/functional/test_basic.py

```diff
@@ -476,6 +476,17 @@ def test_invalid_python(self):
         self.assertIn('1-%s:MyTest.test_my_name -> TestError' % test,
                       result.stdout)
 
+    @unittest.skipIf(process.system("which /bin/read", ignore_status=True),
```

Comment:
sure, let me add this method to `avocado.utils.process` and then reuse it in those tests.